lib/Doctrine/Migrations/Configuration/Loader/PHPFileLoader.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Configuration\Loader;
+
+use Doctrine\Migrations\Configuration\Configuration;
+use Doctrine\Migrations\Configuration\Exception\FileNotFound;
+use function assert;
+use function file_exists;
+use function is_array;
+
+/**
+ * @internal
+ */
+final class PHPFileLoader extends AbstractFileLoader
+{
+    /** @var ArrayLoader */
+    private $arrayLoader;
+
+    public function __construct(?ArrayLoader $arrayLoader = null)
```

Comment:
It's obviously a hard dependency, what do you think about making that clear by removing both the

- nullable type declaration and the
- default value

?